gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -3,19 +3,14 @@
 class ManageIQ::Providers::Amazon::CloudManager::RefreshParser < ManageIQ::Providers::CloudManager::RefreshParser
   include ManageIQ::Providers::Amazon::RefreshHelperMethods
 
-  def initialize(ems, options = nil)
+  def initialize(ems, options = Config::Options.new)
     @ems                 = ems
     @aws_ec2             = ems.connect
     @aws_cloud_formation = ems.connect(:service => :CloudFormation)
     @data                = {}
     @data_index          = {}
     @known_flavors       = Set.new
-
-    @options    = options || {}
-    # Default the collection of images unless explicitly declined
-    @options["get_private_images"] = true  unless @options.key?("get_private_images")
-    @options["get_shared_images"]  = true  unless @options.key?("get_shared_images")
-    @options["get_public_images"]  = false unless @options.key?("get_public_images")
```

Comment:
remove these defaults, because they are already the defaults in [settings.yml](https://github.com/ManageIQ/manageiq/blob/faec76397e8e77d65e98432d8d46e38357632832/config/settings.yml#L105-L107)